src/_includes/macros/util/format/datetime.html

```diff
@@ -0,0 +1,77 @@
+{# ==========================================================================
+
+   format_time()
+
+   ==========================================================================
+
+   Description:
+
+   Format a time for displaying in the format of:
+
+    HH:MM AM ZZZ
```

Comment:
Ah ok, noticed it in a few places, be worth cleaning up to make them all the same